core/creator/src/main/java/io/quarkus/creator/phase/nativeimage/NativeImagePhase.java

```diff
@@ -390,6 +390,8 @@ public void provideOutcome(AppCreator ctx) throws AppCreatorException {
             if (additionalBuildArgs != null) {
                 command.addAll(additionalBuildArgs);
             }
+            command.add("--initialize-at-build-time=");
+            command.add("-H:+ReportExceptionStackTraces");
```

Comment:
We have the same line conditioned by `if (reportExceptionStackTraces) {` a few lines below. I think we should just change the default value of `reportExceptionStackTraces` to `true`.

Reply:
OK I'll make that change.

Reply:
Fixed